frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -156,159 +140,139 @@ class DashboardInner extends Component {
   };
 
   onAddQuestion = () => {
-    const { dashboard } = this.props;
-    this.setEditing(dashboard);
+    this.setEditing(this.props.dashboard);
     this.props.toggleSidebar(SIDEBAR_NAME.addQuestion);
   };
 
-  renderContent = () => {
-    const { dashboard, selectedTabId, isNightMode, isFullscreen } = this.props;
-
-    const canWrite = dashboard?.can_write ?? false;
-
-    const dashboardHasCards = dashboard?.dashcards.length > 0 ?? false;
+  shouldRenderAsNightMode() {
+    return this.props.isNightMode && this.props.isFullscreen;
+  }
 
+  renderContent = () => {
+    const p = this.props;
```

Comment:
Everything that's happening here is removing destructured props as they are essentially slag.